[FEAT] 메인페이지-상단 툴 바 UI

edw216 opened the pull request:
### 관련이슈
- #13 

### 작업내용
- 상단 툴 바 UI 구현
- 내비게이션뷰 UI 구현
- 내비게이션뷰 내부 확장 리스트뷰 구현


### 특이사항
- 바텀 앱 바 -> 상단 툴 바 변경

### 작업시간
- 3d 3h 0m

Commit:
refact(#6) : constraintsLayout 배치 수정
recyclerView 폴더 생성 및 이동

Commit:
fact(#13) : 메인화면 상단 툴바 UI 구현
툴바 내비게이션뷰 UI 구현

MunSeongKim reviewed (dismissed):
패키지 리팩토링 이슈때문에 일시 승인합니다

getch37 committed:
Merge branch 'develop' into feature/#13-BottomAppBar

MunSeongKim committed:
refact(#13): 충돌 이슈 해결

- 충돌 이슈 해결하면서 MainPage/MainPageCalendarActivity.java 파일 데이터 손상 있을 수 있으니 확인 필요합니다.

MunSeongKim commented:
리팩토링 이슈로 인해 일시 클로징 합니다.
